pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -348,6 +348,13 @@ func (c *ClusterUpConfig) makeKubeletFlags(out io.Writer, nodeConfigDir string)
 		flags = append(flags, "--cgroup-driver=cgroupfs")
 	}
 
+	// TODO: OSX snowflake
+	// Default changed in kube 1.10. This ultimately breaks Docker For Mac as every hostPath
+	// mount must be shared or rslave.
+	if runtime.GOOS == "darwin" {
+		flags = append(flags, "--feature-gates=MountPropagation=false")
```

Comment:
@sjenning is feature going to flaggable forever?  If we couldn't set this to false, `oc cluster up` on make would be broken.

Reply:
@jsafrane what is the stable plan for MountPropagation? would there be a way to disable it, or will it just be unconditionally enabled?

Reply:
The plan was to follow usual process - remove the flag after GA. We could add some check to kubelet startup to fall back to Private propagation on Darwin and throw errors when user explicitly requests something else.

Reply:
@jsafrane kubelet runs in VM on OSX which is standard linux... problem is that linux is not really "standard" and does not allow modification of shared mounts. I'm not sure if we can detect that kubelet runs inside Docker VM on Mac. (maybe some snowflake env variable to force it to use private propagation will do?)